runtime/battery-station/src/parameters.rs

```diff
@@ -365,11 +365,33 @@ parameter_types! {
 }
 
 parameter_type_with_key! {
-    // Well, not every asset is a currency ¯\_(ツ)_/¯
+    // Existential deposits used by orml-tokens.
+    // Only native ZTG and foreign assets should have an existential deposit.
+    // Winning outcome tokens are redeemed completely by the user, losing outcome tokens
+    // are cleaned up automatically.
+    // PoolShares are also completely redeemed by the user.
+    // Explicit match arms are used to ensure new asset types are respected.
```

Comment:
I noticed that polkadot-js displays the pool shares even after withdrawing liquidity, just with a zero amount. So I dug a little deeper: https://github.com/open-web3-stack/open-runtime-module-library/blob/4f5a3f34d8cbd98b7bc2e295219a3e7b99b9ecaf/tokens/src/lib.rs#L735-L748.

Turns out that if you set zero as existential deposit, the account _never_ gets reaped. So I think the existential deposit that you want is actually `1`, not `0` (account gets reaped if there's nothing in it).

Reply:
e9edcf419a4065130815146a385184249102f036